numpy/core/tests/test_multiarray.py

```diff
@@ -50,13 +47,10 @@
 from datetime import timedelta, datetime
 
 
-if sys.version_info[:2] > (3, 2):
-    # In Python 3.3 the representation of empty shape, strides and sub-offsets
-    # is an empty tuple instead of None.
-    # https://docs.python.org/dev/whatsnew/3.3.html#api-changes
-    EMPTY = ()
-else:
-    EMPTY = None
+# In Python 3.3+ the representation of empty shape, strides and sub-offsets
+# is an empty tuple instead of None.
+# https://docs.python.org/dev/whatsnew/3.3.html#api-changes
+EMPTY = ()
```

Comment:
Can we eliminate this entirely and just inline `()` below?

Reply:
I mention that I'd like to do this in a follow up PR (which can sometimes make these easier to read) do you mind if I punt on this one for a couple of days (it has +8 uses in both files).

Reply:
My fear is that this type of thing can get left behind, and is harder to cleanup once the context of the `version_info` check is gone.

Some options, in rough preference order:
* Make the PR that cleans this up first, so that this PR is just removing an unused `EMPTY` variable
* Leave an in-source comment here indicating that it's to be cleaned up later
* Create a tracking issue where we can just paste links to this type of thing, and tick them off as they're done
* do the cleanup in the same PR
* merge it anyway

None of these options are unacceptable.

Reply:
I'm happy to try and do the PR that cleans stuff up first (well I'll really have two PRs in parallel but I can hope they get merged in the simplified order) and I'll document in a tracking issue all the things I want to investigate as well.

Reply:
This was resolved by breaking out these changes first in #15307 and adding a issue to track outstanding work